neat-screen.js

```diff
@@ -84,165 +95,246 @@ function NeatScreen (cabal) {
     else return
     this.bus.emit('render')
   })
+
+  // move between window panes with ctrl+j
+  this.neat.input.on('alt-n', () => {
```

Comment:
alt-n still doesnt work for me on mac terminal ... but we can solve it after this lands